src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -280,7 +286,15 @@ private static void AddLinuxContainerServices(this IServiceCollection services)
             services.AddSingleton<IMetricsPublisher>(s =>
             {
                 var environment = s.GetService<IEnvironment>();
-                if (environment.IsFlexConsumptionSku())
+                if (environment.IsV1LinuxConsumptionOnLegion())
```

Comment:
I think we should rename this env helper to IsLinuxConsumptionOnLegion, which matches IsLinuxConsumptionOnAtlas

Reply:
Done.